rdeprojector/controllers/rdeprojector_controller.go

```diff
@@ -60,3 +67,27 @@ func (r *RDEProjectorReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&capvcdv1alpha1.RDEProjector{}).
 		Complete(r)
 }
+
+func (r *RDEProjectorReconciler) reconcileNormal(ctx context.Context, rdeProjector *capvcdv1alpha1.RDEProjector) (ctrl.Result, error) {
+	log := ctrl.LoggerFrom(ctx)
+
+	//TODO Ensure sysadmin persona works
+	workloadVCDClient, err := vcdsdk.NewVCDClientFromSecrets(rdeProjector.Spec.Site, rdeProjector.Spec.Org,
+		"", rdeProjector.Spec.Org, rdeProjector.Spec.UserCredentialsContext.Username,
```

Comment:
We need the same userorg method here. Maybe that should become a common function in CPI common